.github/workflows/ci.yml

```
@@ -1,31 +1,86 @@
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+      - name: Run yarn audit
+        run: yarn audit
+
+  dependency-review:
+    name: Dependency Review
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request'
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/dependency-review-action@v3
```

actionlint (1.7.4)

the runner of "actions/dependency-review-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)